### PR TITLE
use the stack env in the bucket name

### PR DESCRIPTION
**Pull request by ametaireau**
r? @Natim @leplatrem @deanwilson

### CODE REVIEW

**Comment by deanwilson** [comment]
R- 
We can determine a bucket name and pass it through from cloudformation -> puppet -> config file rather than have parts of it hard coded in the application.

**Comment by ametaireau** [comment]
@deanwilson updated.

**Comment by leplatrem** [comment]
It looks good, even though I'm not aware what this is for :(

**Comment by ametaireau** [comment]
It enables ops to chose the bucket name from a configuration variable.

File: loop/filestorage/aws.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var CONTENT_TYPE = "application/json";
 function AwsDriver(options, settings, statsdClient) {
   this.statsdClient = statsdClient;
   this._settings = settings || {};
-  this._publicBucket = options.publicBucket || DEFAULT_PUBLIC_BUCKET;
+  this._publicBucket = settings.bucketName || DEFAULT_PUBLIC_BUCKET;
```
Comment (Natim) [inline loop/filestorage/aws.js]: It should be options and not settings here. because you settings are empty below.

**Pull request merged**